code/modules/clothing/outfits/ert.dm

```diff
@@ -48,11 +48,13 @@
 	name = "ERT Commander - High Alert"
 
 	glasses = /obj/item/clothing/glasses/thermal/eyepatch
+	r_hand = /obj/item/storage/briefcase/lasergun_plus
 	backpack_contents = list(/obj/item/storage/box/engineer=1,\
 		/obj/item/melee/baton/loaded=1,\
 		/obj/item/clothing/mask/gas/sechailer/swat=1,\
 		/obj/item/gun/energy/pulse/pistol/loyalpin=1)
```

Comment:
Commander already comes with a sidearm and Energy sword. They don't need a gangster guncase and an additional hold out blaster.

Reply:
Disabler + spare chargeable ammo for when power is out like in blobs, traitors, xenos. 8/10 times ERT is called they need a holdout weapon do to being disarmed do to not having the no drop implant

Reply:
Red alert ERT have access to the armory, put this shit in there instead of on the ERT loadout then.